src/main/java/com/kodilla/ecommercee/domain/GroupDto.java

```diff
@@ -1,4 +1,4 @@
 package com.kodilla.ecommercee.domain;
 
 public class GroupDto {
```

Comment:
Ten plik chyba nie powinien w ogóle się znaleźć w tym pull request, bo będziemy to robić w innym zadaniu

Reply:
Racja. Znalazł się bo w poprzednim commicie go uzupełniłem, a w tym usunąłem cały kod. To przez pośpiech i brak wiedzy :D